src/main/rx/rx.c

```diff
@@ -591,7 +596,11 @@ static void readRxChannelsApplyRanges(void)
         const uint8_t rawChannel = channel < RX_MAPPABLE_CHANNEL_COUNT ? rxConfig()->rcmap[channel] : channel;
 
         // sample the channel
+#if defined(USE_RX_MSP_OVERRIDE)
```

Comment:
I would change this to the following to make it more obvious that override is off by default:

```
#if defined(USE_RX_MSP_OVERRIDE)
        if (rxConfig->msp_override_channels) {
￼            uint16_t sample = rxMspOverrideReadRawRc(&rxRuntimeState, rawChannel);
        } else
￼#endif
        {
￼            uint16_t sample = rxRuntimeState.rcReadRawFn(&rxRuntimeState, rawChannel);
        }
```

Reply:
I agree, that would be a lot better